Makefile

```diff
@@ -23,4 +27,16 @@ vendor:
 test: vendor
 	go test ./pkg/...
 
+images: asb-image
```

Comment:
I can do that, I was going to just use `make images` and for a second thought we might create more than one. 

What about 

```
images: asb-images apb-image

asb-images:
    ansible-container stuff

apb-image:
    docker build stuff

.PHONY asb-images apb-image
```